episodes/silent-spring.njk

```diff
@@ -2,7 +2,9 @@
 date: 2020-11-02
 stuff:
   title: >
-    Episode 1: Silent Spring
+    Season 1, Episode 1: Silent Spring
+  pageTitle: >
+    Season 1, Episode 1: Silent Spring
```

Comment:
Can we derive this from `episode`, `season`, and `title` instead of having another property?

Reply:
done! added a new filter